XLS-66: Principal-only Vault/LoanBroker accounting under LendingProtocolV1_1 (#582)
XRPLF/XRPL-Standards: XLS-0066-lending-protocol/README.md

Reviewer: ```suggestion
2. Increment `AccountRoot(Borrower).OwnerCount` by `3`.
    1. In XLS-65.1, this was `2`.
    1. In XLS-65.0, this was `1`.

```

Reviewer: ```suggestion
## A-1 Changelog

### A-1.1 `LendingProtocol`: released 2026-01-32

This amendment releases the feature

### A-1.2 `fixCleanup3_1_3`: released 2026-some-date

This amendment makes XYZ minor changes

### A-1.3 `fixCleanup3_2_0`: released 2026-06-something

This amendment makes ABC minor changes

## A-2 F.A.Q.
```

23. The `LoanBroker.Owner` is not authorized for the asset. (`tecNO_AUTH`)
24. The `LoanBroker.LoanSequence` has reached its maximum value. (`tecMAX_SEQUENCE_REACHED`)

##### 3.8.5.3 Protocol-Level Failures (`LendingProtocolV1_1`)

> 

Reviewer 2: Should add updated check 14 since the interest is no longer tracked by AssetsTotal:

    Vault.AssetsMaximum != 0 and Vault.AssetsTotal > Vault.AssetsMaximum (expected interest would exceed vault assets cap). (tecLIMIT_EXCEEDED)

>    - Add `LoanID` to the `OwnerDirectory` of the `LoanBroker` _pseudo-account_ (sets `LoanBrokerNode`).
>    - Add `LoanID` to the `OwnerDirectory` of the `Borrower` (sets `OwnerNode`).

Reviewer: ```suggestion
8. Directory linking:
- Add `LoanID` to the `OwnerDirectory` of the `LoanBroker` _pseudo-account_ (sets `LoanBrokerNode`).
- Add `LoanID` to the `OwnerDirectory` of the `Borrower` (sets `OwnerNode`).
- Add `LoanID` to something else (added in `LendingProtocolV1_1`).
```
